README.md

```diff
@@ -1 +1,56 @@
-# sodym
\ No newline at end of file
+# sodym
+The sodym package provides key functionality for material flow analysis, with the class `MFASystem` acting as a template (parent class) for users to create their own material flow models.
+
+The concepts used in sodym are based on:
+ODYM
+Copyright (c) 2018 Industrial Ecology
+author: Stefan Pauliuk, Uni Freiburg, Germany
+https://github.com/IndEcol/ODYM
+
+## Install
+Clone the sodym repository using git. Activate the virtual environment and install poetry, and then run 'python -m poetry install' to obtain all the necessary dependencies.
```

Comment:
Links to installation instructions for virtual environments and poetry would be nice.

Reply:
I am hoping we can update this soon to "pip install sodym"! Ideally this should also install all dependencies (I haven't checked how it works with poetry, but actually we could have a standard requirements.txt for sodym I guess...) Then cloning the repo will only be necessary if people want to actively develop sodym, rather than just use it. At that stage, we could consider adding a "contribute" section to the readme.

Reply:
Then we can also add the "poetry install --with test,docs" for contributors. Not sure if we want to do this now, or rather wait until with have the pypi option for non-developing users.

Reply:
Fair!